app/frontend/wallet/shelley/helpers/addresses.ts

```diff
@@ -39,8 +39,8 @@ export const encodeAssetFingerprint = (policyIdHex: HexString, assetNameHex: Hex
   return bech32.encode('asset', data)
 }
 
-export const encodeCatalystVotingKey = (votingKey: HexString): string => {
-  return bech32.encode('ed25519_pk', Buffer.from(votingKey, 'hex'))
+export const encodeCatalystVotingKey = (votingKey: HexString | null): string => {
```

Comment:
I would rather make this function accept only a defined votingKey and let the upstream logic handle the case when it isn't (if it can actually happen at all)